### PR TITLE
Add FAQ for the Builder

### PR DESCRIPTION
**Pull request by ericschallock**
* We have loads of questions about the Builder. This answers some of them.

### CODE REVIEW

File: _posts/general/2019-03-19-builder-faq.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+date: 2019-03-19
+title: Builder FAQ
+redirect_from:
+  - /docs/faq
```
Comment (nearnshaw) [inline _posts/general/2019-03-19-builder-faq.md]: These redirects aren't necessary
We only use them when an old page is renamed

Comment (ericschallock) [reply _posts/general/2019-03-19-builder-faq.md]: Whoops. Cool.

File: _posts/general/2019-03-19-builder-faq.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  - /docs/builder-frequently-asked-questions
+description: Frequently Asked Questions for the Decentraland Builder
+categories:
+  - Decentraland
+  - Builder
```
Comment (nearnshaw) [inline _posts/general/2019-03-19-builder-faq.md]: These 'categories' relate to the tree structure, not to search.  I think that adding a new "Builder" category could have unwanted consecuences. I'd just keep the "Decentraland" category

Comment (ericschallock) [reply _posts/general/2019-03-19-builder-faq.md]: Perfect. Thanks! I just didn't understand how the new metadata is set up :(

Comment (ericschallock) [reply _posts/general/2019-03-19-builder-faq.md]: But now I do!

File: _posts/general/2019-03-19-builder-faq.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+type: Document
+set: general
+set_order: 1
```
Comment (nearnshaw) [inline _posts/general/2019-03-19-builder-faq.md]: We don't want this appearing before 'introuduction' or the general 'faq'
Let's set this number to 2

**Pull request merged**